Architecture.md

```diff
@@ -0,0 +1,100 @@
+# Architecture
+
+## High Level
+
+| Entry Point | Game Loop | Outputs        |
+|-------------|-----------|--------------: |
+| main.js     | game.js   | SIG_TERM, Save |
+
+### `main.js`
+
+Serves as a boostrapping file for the environment the game will run in.
```

Comment:
`Serves as a boostrapping file for the environment the game will run in.` ->
`Boostrapping file for the environment the game will run in.`